### ISSUE TITLE
Android Support

### ISSUE THREAD

**Issue by MVoloshin** [barotto/IBMulator#21]
If IBMulator uses SDL for video output, why not to add support for running on Android?

**Comment by barotto** [comment]
Yes it uses SDL and OpenGL 3, so it should be relatively easy to port to Android and macOS (and I'm also a fan of multiplatform software).
Unfortunately I lack the knowlegde, the tools, and most of all the time to do any platforms other than Windows and Linux.
Contributions are welcome!

**Comment by MVoloshin** [comment]
If you add libSDL 1.2 support and remove libRocket dependency, I think it will be possible to easily build IBMulator for Android with this toolchain: https://github.com/pelya/commandergenius/
